docs/user/gcp/install_xpn.md

```diff
@@ -0,0 +1,167 @@
+
+```
+platform:
+  gcp:
+    computeSubnet: <compute-subnet>
+    controlPlaneSubnet: <control-plane-subnet>
+    network: <network>
+    networkProjectID: <networkProjectID>
+```
+
+The installer service account requires the following permissions on the network
+in the Host Project in order to enable the cluster to use the network resources.
+
+- Compute Network User
+
+## Configure cloud firewall rules
+
+Because firewall rules are attached to the network, they must exist in the Host
+Project. As a result, additional permissions would be necessary to manage these
+rules. Choose one of the options below based on if you wish to enable these
+permissions:
+
+### Option A (Default): Automatic firewall rules (requires permissions)
+
+The default behavior is to automatically create and manage firewall rules. This
+option requires the installer service account to have the following permissions
+in the Host Project:
+
+- Compute Network Admin
+- Compute Security Admin
+
+### Option B: Manually created, pre-exsiting firewall rules
+
+When these permissions are not desired, it is possible to create firwall rules
+in advance and configure the installer to skip their creation. It is important
+to ensure the rules are sufficient to enable all cluster communications.
+Creation of firewall rules can be disabled by adding the following to the
+install-config.
+
+```
+platform:
+  gcp:
+    createFirewallRules: Disabled
+```
+
+The pre-existing rules can be cofigured specifically for the cluster instances
+by using network tags in combination with the following tags paremeters in the
+install-config:
+
+```
+compute:
+- name: worker
+  platform:
+    gcp:
+      tags:
+      - mycomputetag
+controlPlane:
+  platform:
+    gcp:
+      tags:
+      - mycontrolplanetag
+platform:
+  gcp:
+    defaultMachinePlatform:
+      tags:
+      - mydefaulttag
+```
+
+## Configure Public DNS Zone
+
+The public DNS zone can exist in either the Service Project or any other
+project. Because the public zone is authorative, it is common for it to exist
+in the Host Project.  As a result, when the public DNS Zone is not in the
+Service Project, additional premissions would be necessary to manage these
+resources. Choose one of the options below based on where your public DNS Zone
+exists, and if you wish to enable these permissions:
+
+### Option A (Default): Automatic in Service Project
+
+The default behavior is to find the DNS Zone in the Service Project and use it
+to create and manage DNS records. No special permissions are required.
+
+### Option B: Automatic in Host Project (Requires Permissions)
+
+Use the following install-config parameters to configure the cluster to use a
+DNS Zone in the Host Project.
+
+```
+platform:
+  gcp:
+    publicDNSZone:
+      id: my-public-zone-id
+      project: my-project-id
+
+```
+
+This option requires the installer service account to have the following
+permissions in the specified project:
+
+- DNS Administrator
```

Comment:
nit: Should we add the link to the information here? (https://cloud.google.com/compute/docs/access/iam#dns.admin)